yt_dlp/downloader/fragment.py

```diff
@@ -324,7 +336,42 @@ def _prepare_external_frag_download(self, ctx):
             'fragment_index': 0,
         })
 
-    def download_and_append_fragments(self, ctx, fragments, info_dict, *, pack_func=None, finish_func=None):
+    # ctx1, fragments1, info_dict1, ctx2, fragments2, info_dict2, ...
+    # length of arguments must be multiple of 3, or you'll see broken console
+    def download_and_append_fragments_multiple(self, *args, pack_func=None, finish_func=None):
```

Comment:
not good design. better to let args be (a list of) dicts/tuples

Reply:
okay